src/_posts/2020-10-28-cass_changelog_1.markdown

```diff
@@ -0,0 +1,92 @@
+
+## In the News
+
+**Solutions Review** - [The Five Best Apache Cassandra Books on Our Reading List](https://solutionsreview.com/data-management/the-five-best-apache-cassandra-books-on-our-reading-list/)
+
+**ZDNet** - [What Cassandra users think of their NoSQL DBMS](https://www.zdnet.com/article/what-cassandra-users-think-of-their-nosql-dbms/)
+
+**Datanami** - [Cassandra Adoption Correlates with Experience](https://www.datanami.com/2020/09/22/cassandra-adoption-correlates-with-experience/)
+
+**Container Journal -** [5 to 1: An Overview of Apache Cassandra Kubernetes Operators](https://containerjournal.com/topics/container-management/5-to-1-an-overview-of-apache-cassandra-kubernetes-operators/)
+
+**Datanami** - [Cassandra Gets Monitoring, Performance Upgrades](https://www.datanami.com/2020/07/21/cassandra-gets-monitoring-performance-upgrades/)
+
+**ZDNet** - [Faster than ever, Apache Cassandra 4.0 beta is on its way](https://www.zdnet.com/article/faster-than-ever-apache-cassandra-4-0-beta-is-on-its-way/)
+
+## Cassandra Tutorials & More
+
+A Cassandra user was in search of a tool to perform schema DDL upgrades. Another user suggested [https://github.com/patka/cassandra-migration](https://github.com/patka/cassandra-migration) to ensure you don&#39;t get schema mismatches if running multiple upgrade statements in one migration. See the [full email](https://lists.apache.org/thread.html/rdfee145c4c5d920f644c6bcd081c6fb446d52b055c133485217b8143%40%3Cuser.cassandra.apache.org%3E) on the user mailing list for other recommended tools.
+
+[Start using virtual tables in Apache Cassandra 4.0](https://opensource.com/article/20/10/virtual-tables-apache-cassandra) - Ben Bromhead, Instaclustr
+
+[Benchmarking Apache Cassandra with Rust](https://pkolaczk.github.io/benchmarking-cassandra/) - Piotr Kołaczkowski, DataStax
+
+[Open Source BI Tools and Cassandra](https://blog.anant.us/open-source-bi-tools-and-cassandra/) - Arpan Patel, Anant Corporation
+
+[Build Fault Tolerant Applications With Cassandra API for Azure Cosmos DB](https://dzone.com/articles/build-fault-tolerant-applications-with-cassandra-a) - Abhishek Gupta, Microsoft
+
+[Understanding Data Modifications in Cassandra](https://www.red-gate.com/simple-talk/blogs/understanding-data-modifications-in-cassandra/) - Sameer Shukla, Redgate
+
+## 
+![Cassandra Changelog footer](/img/changelog_footer.jpg "image_tooltip")
+## 
+Cassandra Changelog is curated by the community. Please send submissions to [cassandra@constantia.io](mailto:cassandra@constantia.io).
```

Comment:
> Content submissions are directed to cassandra@constantia.io. @michaelsembwever we should probably note this in the repo README so we know to update the blog pages should it change in the future.

I don't know what the best approach is here. 

A separate mailing list I think would be best. For example media@cassandra.apache.org but I am open to suggestions to the name.  We can do this later on, we'd need some agreement on the dev ML first.
@loganloganlogan what are your thoughts on this?